mscore/pagesettings.cpp

```diff
@@ -138,24 +133,37 @@ void PageSettings::blockSignals(bool block)
 
 void PageSettings::updateValues()
       {
-      Score* sc = preview->score();
-      bool mm = mmButton->isChecked();
-
       blockSignals(true);
 
-      const char* suffix;
+      double converter = convertBy();
+
+      Score*      sc = preview->score();
+      QString suffix = unitSuffixes[int(pf->units())];
+
       double singleStepSize;
       double singleStepScale;
-      if (mm) {
-            suffix = "mm";
-            singleStepSize = 1.0;
-            singleStepScale = 0.2;
-            }
-      else {
-            suffix = "in";
-            singleStepSize = 0.05;
+      switch (pf->units()) {
+      case Units::PX :
+            singleStepSize  = 1; // no fractional points/pixels for now
+            singleStepScale = 1;
+            pxButton->setChecked(true);
+            break;
+      case Units::INCH :
+            singleStepSize  = 0.05;
             singleStepScale = 0.005;
-            }
+            inchButton->setChecked(true);
+            break;
+      default : // Units:MM - the default since the inception of MuseScore
+            singleStepSize  = 1.0;
+            singleStepScale = 0.2;
+            mmButton->setChecked(true);
+            break;
+      }
+
+      pageWidth->setSuffix(suffix);
```

Comment:
Yes, none of the suffixes displayed in the Page Settings dialog are translated, and never were AFAIK.